setup.cfg

```diff
@@ -27,7 +27,7 @@ install_requires =
     hypothesis
     selenium
     playwright
-    tblib
+    pyodide-tblib
```

Comment:
Could you add a brief comment here about why we use a forked version of tblib, so we don't forget this later.